website/settings/defaults.py

```diff
@@ -281,6 +281,7 @@ def parent_dir(path):
 
 # Most Popular and New and Noteworthy Nodes
 POPULAR_LINKS_NODE = None  # TODO Override in local.py in production.
+POPULAR_LINKS_NODE_REGISTRATIONS = None  # TODO Override in local.py in production.
```

Comment:
Minor, but let's change references to `node_registrations` just to `registrations`, so that this becomes `POPULAR_LINKS_REGISTRATIONS` and in the other 25ish places where similar things are done. Since all registrations are of nodes, I think this is less redundant? Unless I'm missing something.